pkg/app/app.go

```diff
@@ -390,26 +390,29 @@ func initClients(c *config.Config) error {
 	// method on the app. Forces developers at authorship time to think about
 	// dependencies / make sure things are ready.
 	log.Notice("Initializing clients...")
-	log.Debug("Trying to connect to etcd")
 
-	// Intialize the etcd configuration
-	clients.InitEtcdConfig(c)
-	etcdClient, err := clients.Etcd()
-	if err != nil {
-		return err
-	}
+	if strings.ToLower(c.GetString("dao.type")) != "crd" {
 
-	ctx, cancelFunc := context.WithCancel(context.Background())
-	defer cancelFunc()
+		log.Debug("Trying to connect to etcd")
+		// Intialize the etcd configuration
+		clients.InitEtcdConfig(c)
+		etcdClient, err := clients.Etcd()
+		if err != nil {
+			return err
+		}
```

Comment:
:+1: took a while to figure out what really changed here, but it was the if statement that caused the indentation of the block of code. looks good.